pajbot/tmi.py

```diff
@@ -1,8 +1,33 @@
+from enum import Enum
+
+
+class WhisperOutputMode(Enum):
+    DISABLED = 0
+    NORMAL = 1
+    CHAT = 2
+
+    @staticmethod
+    def from_config_value(config_value):
+        return {
+            "disabled": WhisperOutputMode.DISABLED,
+            "normal": WhisperOutputMode.NORMAL,
+            "chat": WhisperOutputMode.CHAT,
+        }.get(config_value.lower(), WhisperOutputMode.NORMAL)
```

Comment:
Last discussion: Is this behaviour good? Should an invalid config option (e.g. a missspelling) be silently swallowed?

Reply:
no, i think an invalid config option should throw a warning and then revert to normal behavior.

Reply:
I agree with RAnders, since we already grab a default value "normal" if no value is set, adding a crash on misconfiguration seems fine. Adding the crash adds no extra friction for already created bots

Reply:
👍